vendor/bat-native-ads/src/bat/ads/internal/creatives/embeddings_database_table.cc

```diff
@@ -0,0 +1,128 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/creatives/embeddings_database_table.h"
+
+#include <functional>
```

Comment:
not used, can you manually check system includes and remove any we do not use from the PR please, thanks